algorithmic_efficiency/workloads/imagenet/imagenet_pytorch/workload.py

```diff
@@ -19,18 +22,10 @@
 from algorithmic_efficiency.workloads.imagenet.workload import \
     BaseImagenetWorkload
 
-DEVICE = torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
-
-
-# from https://github.com/pytorch/pytorch/issues/23900#issuecomment-518858050
-def cycle(iterable):
-  iterator = iter(iterable)
-  while True:
-    try:
-      images, labels = next(iterator)
-      yield {'inputs': images, 'targets': labels}
-    except StopIteration:
-      iterator = iter(iterable)
+PYTORCH_DDP = 'LOCAL_RANK' in os.environ
```

Comment:
nit: should we rename this `USE_PYTORCH_DDP` ?

Reply:
`USE_PYTORCH_DDP` sounds good to me